src/ucp/core/ucp_request.h

```diff
@@ -52,6 +53,17 @@ enum {
         } \
     }
 
+/*
+ * - Mark the request as completed
+ * - Otherwise - the request might be released - if so, return it to mpool.
+ */
+#define ucp_request_rma_complete(_req) \
```

Comment:
The other one call callback function that I don't need for RMA. If we merge the two macros it introduces branch in the critical path.